lab-andrew/route/brew-route.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const Router = require('express').Router;
+const jsonParser = require('body-parser').json();
+const brewCrud = require('../lib/brew-crud');
+const AppError = require('../lib/app-error');
+const brewRouter = module.exports = new Router();
+
+brewRouter.post('/brew', jsonParser, function(req, res) {
+  brewCrud.createBrew(req.body)
+  .then(brew => res.send(brew))
+  .catch(err => res.sendError(err));
+});
+
+brewRouter.get('/brew', function(req, res) {
```

Comment:
This route actually doesn't work, and you didn't test for it in your tests.
This needs to be '/brew/:id'